clang/lib/Driver/Driver.cpp

```diff
@@ -7882,9 +7882,9 @@ Action *Driver::BuildOffloadingActions(Compilation &C,
         break;
       }
 
-      // Backend/Assemble actions are not used for the SYCL device side
-      if (Kind == Action::OFK_SYCL &&
-          (Phase == phases::Backend || Phase == phases::Assemble))
+      // Assemble actions are not used for the SYCL device side.  Both compile
```

Comment:
Is this  going to deviate from the old-offloading-model flow where we do not seem to be calling Backend? If yes, is that agreeable? Thanks

Reply:
Yes, this is agreeable.  The functionality is the same, we just introduce the additional phase to better match with the new model expectations.